ksql-functional-tests/src/test/resources/query-validation-tests/timestampformat.json

```
@@ -37,6 +37,38 @@
       "outputs": [
         {"topic": "TS", "value": {"ETS": 1566912669200}, "timestamp": 1566912669200}
       ]
+    },
+    {
+      "name": "KSQL override output timestamp for CSAS",
```

nit: `KSQL` in name is superfluous!

```suggestion
      "name": "override output timestamp for CSAS",
```

---

Done